### DISCUSSION TITLE
Stock Assessment Prioritization

### DISCUSSION THREAD

**Discussion by rebecca-vanhoeck** [SAEL-SWFSC/PAM-SI-Mtg-March2024#87] [Q&A]
Of the following three projects for PAM Data Integration for Stock Assessments, in what order would you rank their priority for your group? 

A. Improving Automation of Sperm Whale/Beaked Whale Density Estimation from towed arrays
B. Incorporating glider data into an SDM Framework
C. Incorporating stationary data into an SDM Framework

**Comment by shannonrankin** [comment]
B
C
A

**Reply by namariD** [reply]
C
A
B

**Comment by jlkeating** [comment]
A, B, C

**Comment by eoleson** [comment]
B
A
C

**Comment by ludovictenorio** [comment]
C
B
A

**Comment by cberchok** [comment]
C would be our priority  with 2 caveats
1. We don't use gliders or towed arrays , but if we ever did have those capabilities, A and B would be equally good.
2. Although C is our current priority, we are already working with a statistician (Paul Conn) to do just this with bearded seal data, so this would not be a priority for your group for us.

**Comment by ybarkley** [comment]
B, A, C

**Comment by marlaholt** [comment]
@shannonrankin test

**Comment by rebecca-vanhoeck** [comment]
Current prioritization as of 3/22 at noon: 

B: Glider SDM (15 points)
C: Stationary SDM (14 points)
A: Density estimation from towed array (13 points)

This ranking was based on assigning 3 points for being ranked 1st, 2 points for being ranked 2nd, and 1 point for being ranked 3rd.

**Comment by sofievanparijs** [comment]
A, C, B since I think this is the most realistic order in which these can be achieved.
